docs/BR.md

```diff
@@ -2119,14 +2141,16 @@ These methods allow domain owners to publish contact information in DNS for the
 
 ### B.1.1. CAA contactemail Property
 
-SYNTAX: contactemail <rfc6532emailaddress> 
+SYNTAX: contactemail <rfc6532emailaddress>
```

Comment:
Do you need to escape this syntax with backticks? That is, does it try to make this a link?

Reply:
It doesn't try to make a link, but it does just ignore the \<\> contents, which isn't good. (See screenshot below) If we surround it with backticks, we'll get that whole line literally--is that the right output?
![br_sc26_sc27_b111](https://user-images.githubusercontent.com/1791326/76449502-9393a500-63a2-11ea-9d54-ac082423e003.png)

Reply:
```suggestion
SYNTAX: `contactemail <rfc6532emailaddress>`
```

Reply:
Good! Fixed, and changed the same block in B.1.2 to match (in f4f2597).